rediscluster/client.py

```diff
@@ -53,6 +53,50 @@
     TimeoutError,
 )
 
+# Not complete, but covers the major ones
+# https://redis.io/commands
+READ_COMMANDS = frozenset([
```

Comment:
since I had to move `READ_COMMANDS` to make it available to the pipeline object I ended up converting it into a frozenset (stolen from https://github.com/Grokzen/redis-py-cluster/pull/406) since it does speed up lookups by an order of magnitude 😮